spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -512,6 +512,12 @@ def seatle_style arg = default(42)
       RUBY
     end
 
+    it 'accepts parens in range literals' do
+      expect_no_offenses(<<-RUBY)
+        1..limit(n)
+      RUBY
+    end
```

Comment:
Looks good! On the other hand, there is also `erange` in the similar case :-)

```ruby
1...limit(n)
```

Would you please add along with the CHANGELOG entry?

Reply:
Oh, didn't know that. Thanks for the catch!